LICENCE.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- See http://creativecommons.org/licenses/by-sa/4.0/legalcode -->
+<!-- Human-readable summary: http://creativecommons.org/licenses/by-sa/4.0/ --> 
+<head>
+  <title>Creative Commons — Attribution-ShareAlike 4.0 International — CC BY-SA 4.0</title>
```

Comment:
The RtF Consortium IPR agreement specifically states the Creative Commons Attribution International license for GPII-related projects. So please go ahead and update it to be less restrictive.